modules/test/playwright/tests/cookies-banner-web/cookiesBannerScript.spec.ts

```diff
@@ -22,7 +22,9 @@ test('@LPD-25701 Cookie Banner Script', async ({
 	systemSettingsPage,
 }) => {
 	await test.step('Go to page and click edit', async () => {
-		await page.goto(layout.friendlyURL);
+		await page.goto(layout.friendlyUrlPath);
+
+		await page.locator('#content').waitFor({state: 'visible'});
```

Comment:
Instead of this, can we just await edit button?
```js
const editButton = await page.getByRole('link', {name: 'Edit'});
```

Reply:
This is also flaky it seems. It get stuck not finding the Edit button. Trying to avoid this.